File: codeforces/0004A.cpp
#include <iostream>
#include <string>

int main() {
    std::string input;
    std::getline(std::cin, input);    
    int n = std::stoi(input);
    bool b = n % 2 == 0 && n > 2;
    std::cout << (b ? "YES" : "NO") << std::endl;
    return 0;
}

File: codeforces/0004A.hs
import Text.Read

f :: Int -> String
f x
    | x == 2 = "NO"
    | even x = "YES"
    | otherwise = "NO"

main :: IO ()
main = do
    input <- getLine
    case readMaybe input :: Maybe Int of
        Just number -> putStrLn $ f number
        Nothing -> putStrLn "Invalid input"
